soh/soh/Enhancements/randomizer/randomizer_entrance_tracker.cpp

```diff
@@ -831,6 +831,10 @@ void DrawEntranceTracker(bool& open) {
                     const EntranceData* original = GetEntranceData(entrance.index);
                     const EntranceData* override = GetEntranceData(entrance.override);
 
+                    if (original->type == ENTRANCE_TYPE_DUNGEON || original->type == ENTRANCE_TYPE_GROTTO || original->type == ENTRANCE_TYPE_INTERIOR)
+                        if (original->oneExit != 1 && OTRGlobals::Instance->gRandomizer->GetRandoSettingValue(RSK_DECOUPLED_ENTRANCES) == RO_GENERIC_OFF)
+                            continue;
```

Comment:
We generally do not like bracket-less if statements, as they can introduce errors pretty easily. This can be done with one if statement overall (the expression can be on multiple lines).

And can you add a comment clarifying what this is doing.

Also this should be moved in the loop section above as thats used for filtering and will populate the underlying `displayEntrances` array used in this loop (this will also make sure the "undiscovered" count when using "Hide undiscovered" is reported correctly).

Reply:
Ah, yes, that loop would make more sense. If I were to make a setting for it, what should I put in as the text? I was thinking "Show Redundant Entrances", but while it's descriptive, it seems unnecessarily long.

Reply:
Maybe "Hide reverse", with a tooltip that says "Hide reversed entrances for two-way entrances when decoupled entrances is turned off".

We can make the default for the CVar be on so that its defaulted to true, and then the checkbox can be disabled if decoupled is on. There is a few examples of how to mark a checkbox as disabled in the randomizer.cpp

Reply:
I got the disabling just fine, but since the Imgui element used is the `EnhancementCheckbox`, how do I set the default to on?

Reply:
Nevermind, looks like it just needed a toggle to remember its state across loads.

Reply:
Think it's all done.